ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -628,7 +641,7 @@ export const HelpSidebarIcons = (props: HelpSidebarIconsProps) => {
   );
 
   if (
```

Comment:
We need to continue to display `'apps'` for RStudio here, so I think we need 3 ifs now.  Let's move the access check to the outside...

```suggestion
  if (WorkspacePermissionsUtil.canWrite(workspace.accessLevel)) {
    if (showAppsPanel(config)) {
      keys.push('apps');
    }
    if (config.enableCromwellGKEApp) {
      keys.push('cromwellConfig');
    }
    keys.push('runtimeConfig', 'terminal');
  }
```

Reply:
I feel like I'm being immediately punished for [generalizing tests across apps](https://github.com/all-of-us/workbench/pull/7450#discussion_r1122337334)! The test `should display apps icon for writable workspaces when showAppsPanel is true` would have caught this if we stubbed `showAppsPanel` to be true instead of proxying for it via `enableCromwellGKEApp`.

I modified the test and added a new one that would have caught this.

Reply:
In general, I'm still not sure whether stubbing helper methods or exhausting their parameters is a better testing approach.